client/src/pages/collections/Types.ts

```diff
@@ -29,6 +29,15 @@ export interface CollectionCreateParam {
   description: string;
   autoID: boolean;
   fields: Field[];
+  consistency_level: number;
+}
+
+export enum ConsistencyLevelEnum {
+  Strong = 0,
```

Comment:
we can define Strong = "Strong" .... in enum . 
Then we can pass this value to milvus directly.